test/kubemark/common/util.sh

```diff
@@ -26,24 +26,22 @@ function run-cmd-with-retries {
     if [[ "${ret_val:-0}" -ne "0" ]]; then
       if [[ $(echo "${result}" | grep -c "already exists") -gt 0 ]]; then
         if [[ "${attempt}" == 1 ]]; then
-          # shellcheck disable=SC2154 # Color defined in sourced script
+          # shellcheck disable=SC2154 
```

Comment:
Is there a better way to "import" constant-ish variables in shell that doesn't need this sprinkled all over?

Reply:
> Is there a better way to "import" constant-ish variables in shell that doesn't need this sprinkled all over?

I'll investigate this in time and talk to the shellcheck devs if needs be.